src/Languages/nl-NL.xml

```diff
@@ -2269,18 +2270,18 @@ Probeer het later opnieuw a.u.b.</XDownloadFailed>
   <SplitLongLines>
     <Title>Lange teksten opdelen</Title>
     <SingleLineMaximumLength>Maximale regellengte</SingleLineMaximumLength>
-    <LineMaximumLength>Maximale tekstlengte</LineMaximumLength>
-    <LineContinuationBeginEndStrings>Regelvoortzetting begin/einde tekst</LineContinuationBeginEndStrings>
+    <LineMaximumLength>Maximale titellengte</LineMaximumLength>
+    <LineContinuationBeginEndStrings>Tekens voor in-/uitpunten titels</LineContinuationBeginEndStrings>
     <NumberOfSplits>Afbrekingen: {0}</NumberOfSplits>
     <LongestSingleLineIsXAtY>Langste regellengte is {0} in tekst {1}</LongestSingleLineIsXAtY>
-    <LongestLineIsXAtY>Langste tekstlengte is {0} in tekst {1}</LongestLineIsXAtY>
+    <LongestLineIsXAtY>Langste titellengte is {0} in tekst {1}</LongestLineIsXAtY>

```

Comment:
**`in tekst`** ⇒ **`in titel`** ?

Reply:
In a lot of strings, "line" refers to the whole subtitle. I chose to use "titel" there for clarification.
For example: "Selected lines" was translated as "Tekstselectie", which implies to refer to the selected text, while the source was referring to the currently selected subtitles. In those cases, I translated it as "Geselecteerde titels".